Description: update readme
 .
 syncterm (20140108-1) unstable; urgency=low
 .
   * Initial release (Closes: #nnnn)  <nnnn is the bug number of your ITP>
Author: Fernando Toledo <ragnarok@docksud.com.ar>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- syncterm-20140108.orig/README.md
+++ syncterm-20140108/README.md
@@ -2,3 +2,38 @@ pkg-syncterm
 ============
 
 Debian package for SyncTerm (Synchronet Terminal Emulator)
+
+SyncTERM is a BBS terminal program which supports:
+
+    Windows 95/98/ME/NT/2000/XP, Linux, OpenBSD, NetBSD, OS X, and FreeBSD
+    X/Y/ZModem up/downloads
+    Runs in full-screen mode on ALL platforms (ALT-Enter switches modes)
+    *nix versions will run using SDL, X11, or using curses
+    Full ANSI-BBS support
+    Full CGTerm Commodore 64 PETSCII support
+    Full Atari 8-bit ATASCII support
+    DoorWay support
+    Support for IBM low and high ASCII including the face graphics (☺ and ☻) and card symbols (♥, ♦, ♣, and ♠) which so many other terms have problems with (may not work in curses mode... depends on the terminal being used).
+    Phone books
+    Multiple screen modes (80x25, 80x28, 80x43, 80x50, 80x60, 132x25, 132x28, 132x30, 132x34, 132x43, 132x50, 132x60)
+    ANSI Music (through the sound card if installed)
+    Telnet, RLogin, SSH, RAW, modem, and direct serial connections
+    Auto-login with Synchronet RLogin
+    Large Scrollback
+    Mouse-driven menus
+    Copy/Paste
+    Supports character pacing for ANSI animation as well as the VT500 ESC[*r sequence to allow dynamic speed changes
+    Comes with 37 standard fonts and allows the BBS to change the current font *and* upload custom fonts. This tool will allow you to create fonts for use with SyncTERM.
+    Supports Operation Overkill ][ Terminal emulation
+
+For detailed specs on SyncTERMs handling of ANSI, as well as it's many extensions, refer to this document
+
+To install SyncTERM as your default telnet:// and rlogin:// handler follow the instructions for *nix Mozilla/Firefox or Win32.
+
+Contact me on Synchronix if you have an questions, or requests.
+
+OSSHome.org has provided precompiled binaries for Linux on SPARC systems. Still direct your bug reports to me.
+
+Stable releases are available from the SourceForge.net Logo project page.
+
+You can get the latest CVS code with the following command cvs -d :pserver:anonymous@cvs.synchro.net:/cvsroot/sbbs co syncterm
\ No newline at end of file
